2022-08-18 09:07:07,453 INFO    Thread-2  :97888 [wandb_setup.py:_flush():75] Loading settings from /home/mgatlin3/.config/wandb/settings
2022-08-18 09:07:07,453 INFO    Thread-2  :97888 [wandb_setup.py:_flush():75] Loading settings from wandb/settings
2022-08-18 09:07:07,453 INFO    Thread-2  :97888 [wandb_setup.py:_flush():75] Loading settings from environment variables: {'project': 'sweep-FT', 'entity': 'mgatlin3', 'root_dir': '/home/mgatlin3/FT', 'sweep_id': 'otbrtpw1', 'run_id': 'bfxfaqcq', 'sweep_param_path': '/home/mgatlin3/FT/wandb/sweep-otbrtpw1/config-bfxfaqcq.yaml'}
2022-08-18 09:07:07,453 INFO    Thread-2  :97888 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'sweep.py', 'program': '/home/mgatlin3/FT/sweep.py'}
2022-08-18 09:07:07,453 INFO    Thread-2  :97888 [wandb_init.py:_log_setup():437] Logging user logs to /home/mgatlin3/FT/wandb/run-20220818_090707-bfxfaqcq/logs/debug.log
2022-08-18 09:07:07,453 INFO    Thread-2  :97888 [wandb_init.py:_log_setup():438] Logging internal logs to /home/mgatlin3/FT/wandb/run-20220818_090707-bfxfaqcq/logs/debug-internal.log
2022-08-18 09:07:07,453 INFO    Thread-2  :97888 [wandb_init.py:init():471] calling init triggers
2022-08-18 09:07:07,454 INFO    Thread-2  :97888 [wandb_init.py:init():474] wandb.init called with sweep_config: {'batch_size': 128, 'dropout_c': 0.25, 'dropout_ga': 0.35, 'epochs': 50, 'learning_rate': 0.01, 'num_features_c1': 32, 'num_features_c2': 32, 'num_features_c3': 128, 'num_features_c4': 32}
config: {'batch_size': 4, 'epochs': 5, 'dropout_c': 0.3, 'dropout_ga': 0.1, 'num_features_c1': 64, 'num_features_c2': 64, 'num_features_c3': 128, 'num_features_c4': 64, 'learning_rate': 0.0003}
2022-08-18 09:07:07,454 INFO    Thread-2  :97888 [wandb_init.py:init():524] starting backend
2022-08-18 09:07:07,454 INFO    Thread-2  :97888 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-08-18 09:07:07,469 INFO    Thread-2  :97888 [backend.py:ensure_launched():217] starting backend process...
2022-08-18 09:07:07,476 INFO    Thread-2  :97888 [backend.py:ensure_launched():222] started backend process with pid: 97912
2022-08-18 09:07:07,477 INFO    Thread-2  :97888 [wandb_init.py:init():533] backend started and connected
2022-08-18 09:07:07,484 INFO    Thread-2  :97888 [wandb_run.py:_config_callback():1131] config_cb None None {'batch_size': 128, 'dropout_c': 0.25, 'dropout_ga': 0.35, 'epochs': 50, 'learning_rate': 0.01, 'num_features_c1': 32, 'num_features_c2': 32, 'num_features_c3': 128, 'num_features_c4': 32}
2022-08-18 09:07:07,487 INFO    Thread-2  :97888 [wandb_init.py:init():597] updated telemetry
2022-08-18 09:07:07,503 INFO    Thread-2  :97888 [wandb_init.py:init():628] communicating run to backend with 30 second timeout
2022-08-18 09:07:08,193 INFO    Thread-2  :97888 [wandb_run.py:_on_init():1923] communicating current version
2022-08-18 09:07:08,285 INFO    Thread-2  :97888 [wandb_run.py:_on_init():1927] got version response upgrade_message: "wandb version 0.13.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-08-18 09:07:08,285 INFO    Thread-2  :97888 [wandb_init.py:init():659] starting run threads in backend
2022-08-18 09:07:12,700 INFO    Thread-2  :97888 [wandb_run.py:_console_start():1897] atexit reg
2022-08-18 09:07:12,700 INFO    Thread-2  :97888 [wandb_run.py:_redirect():1770] redirect: SettingsConsole.REDIRECT
2022-08-18 09:07:12,701 INFO    Thread-2  :97888 [wandb_run.py:_redirect():1775] Redirecting console.
2022-08-18 09:07:12,701 ERROR   Thread-2  :97888 [wandb_run.py:_redirect():1834] Failed to redirect.
Traceback (most recent call last):
  File "/home/mgatlin3/.local/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 1827, in _redirect
    out_redir.install()
  File "/home/mgatlin3/.local/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/mgatlin3/.local/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/home/mgatlin3/.local/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/mgatlin3/.conda/envs/poop/lib/python3.9/signal.py", line 56, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread of the main interpreter
2022-08-18 09:07:12,702 INFO    Thread-2  :97888 [wandb_init.py:init():684] run started, returning control to user process
2022-08-18 09:07:12,703 INFO    Thread-2  :97888 [wandb_config.py:__setitem__():150] config set architecture_name = CNN - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f46e30447c0>>
2022-08-18 09:07:12,703 INFO    Thread-2  :97888 [wandb_run.py:_config_callback():1131] config_cb architecture_name CNN None
